app/controllers/concerns/history/log_filter.rb

```diff
@@ -2,21 +2,21 @@ module History::LogFilter
   extend ActiveSupport::Concern
 
   private
-    def put_history_log
-      log = History::Log.new
-      log.url          = request.path
-      log.controller   = params[:controller]
-      log.action       = params[:action]
-      log.user_id      = @cur_user.id if @cur_user
-      log.site_id      = @cur_site.id if @cur_site
+  def put_history_log
```

Comment:
__[rails_best_practices]__
* replace complex creation with factory method (log attribute_assignment_count > 2) [:ref](http://rails-bestpractices.com/posts/2010/07/21/replace-complex-creation-with-factory-method/)
* use model association (for log) [:ref](http://rails-bestpractices.com/posts/2010/07/19/use-model-association/)

<!-- Comment by [SideCI](https://sideci.com) -->